libs/features/personalization/personalization.js

```diff
@@ -271,30 +269,87 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
   return config;
 }
 
-function getPersonalizationVariant(manifestPath, variantNames = [], variantLabel = null) {
+export const loadEntitlements = (() => {
+  let entitlements;
+  return (async () => {
+    if (!entitlements) {
+      const { default: getUserEntitlements } = await import('../../blocks/global-navigation/utilities/getUserEntitlements.js');
+      await loadIms();
+      entitlements = getUserEntitlements();
+    }
+    return entitlements;
+  });
+})();
+
+// remove underscores from entitlement names
+const rmUs = (obj) => Object.entries(obj).reduce((newObj, [key, val]) => {
```

Comment:
could you remove the comment if this was named more robustly? `removeEnttitlementUnderscore`?